website/src/content/concepts/integrate-with-your-own-api.mdx

```diff
@@ -202,6 +208,36 @@ exports.handler = async function (request, response) {
 };
 ```
 
+Other cloud providers might not use a standard Node.js/Express `request` object. For example AWS Lambda functions receive an [`Event` object](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-create-api-as-simple-proxy-for-lambda.html) with different properties.
```

Comment:
```suggestion
Other cloud providers might not use a standard Node.js/Express `request` object. For example AWS Lambda functions receive an [`event` object](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-create-api-as-simple-proxy-for-lambda.html) with different properties.
```

Reply:
Thanks for the suggestion.

Updated in this commit 8991f1078f37d0b2ea7f6e8e8dbc389387c8a64b